libpretixsync/src/main/java/eu/pretix/libpretixsync/sqldelight/AndroidUtilDateAdapter.kt

```diff
@@ -0,0 +1,24 @@
+package eu.pretix.libpretixsync.sqldelight
+
+import app.cash.sqldelight.ColumnAdapter
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.TimeZone
+
+class AndroidUtilDateAdapter : ColumnAdapter<Date, String> {
```

Comment:
Can this adapter be used by JVM as well or is it really android specific?

Reply:
It's specific to how requery saved `Date` values as strings to the DB on Android. While we get `2024-11-13T09:49:28Z` there, it uses a different format on a plain JVM (`2024-11-13 09:49:28.210`).
In theory, you can use this adapter in other places. But you probably want the `JavaUtilDateAdapter` for backwards compatibility on the JVM.